Resources/Private/JavaScript/Terminal/src/helpers/fetchCommands.ts

```diff
@@ -3,17 +3,29 @@ import { fetchWithErrorHandling } from '@neos-project/neos-ui-backend-connector'
 import { CommandList } from '../interfaces';
 
 const fetchCommands = async (endPoint): Promise<{ success: boolean; result: CommandList }> => {
-    return fetchWithErrorHandling
-        .withCsrfToken((csrfToken) => ({
-            url: endPoint,
-            method: 'GET',
-            credentials: 'include',
-            headers: {
-                'X-Flow-Csrftoken': csrfToken,
-                'Content-Type': 'application/json',
-            },
-        }))
-        .then((response) => response && response.json());
+    try {
+        const response = await fetchWithErrorHandling
+            .withCsrfToken((csrfToken) => ({
+                url: endPoint,
+                method: 'GET',
+                credentials: 'include',
+                headers: {
+                    'X-Flow-Csrftoken': csrfToken,
+                    'Content-Type': 'application/json',
+                },
+            }));
+
+        if (response.status === 403) {
```

Comment:
Would be better to check for `response.ok` here and also verify if the json can be parsed and then return it.